wordpress-mobile/WordPress-Android
# Media Async: Visual editor support

Adds support for asynchronously (with the editor closed) updating in-progress media uploads in posts written in the visual editor.

To test:
1. Enable 'Visual' editor
2. Create a post and upload media
3. Exit the editor before the media completes
4. Wait long enough for the media to complete (or watch the logs)
5. Re-open the post in the editor
6. The media item should load as already uploaded

Try above for photos and video, on WordPress.com and self-hosted sites, and on devices above and below API 19. Also make sure the tests run.

cc @mzorz
